webeoc/src/webeoc-scheduler/webeoc-scheduler.service.ts

```diff
@@ -105,23 +105,36 @@ export class WebEocScheduler {
       },
     };
 
+    // default filter grabs complaints that are newer than a specific date
+    const filterItems = [dateFilter];
+
+    // construct the body of the request
     const body = {
       customFilter: {
         boolean: "and",
-        items: [
-          dateFilter,
-          {
-            fieldname: flagName,
-            operator: "Equals",
-            fieldvalue: "Yes",
-          },
-        ],
+        items: filterItems,
       },
     };
 
     try {
       const response = await axios.post(url, body, config);
-      return response.data as Complaint[];
+      const complaints = response.data as Complaint[];
+      let filteredComplaints: Complaint[];
+      if (flagName === WEBEOC_FLAGS.COMPLAINTS) {
```

Comment:
This logic is incorrect.   It will only work for complaints but not Updates, Actions Taken or Updates to Actions Taken.

As a result Complaints are being filtered properly but all other messages are going through regardless of if the flag is set or not.   E.g. if I create an update for a complaint in Victoria, the complaint is properly filtered out but the update is staged (and promptly errors because it can't find the complaint)